source/solvers/fluid_dynamics_matrix_free.cc

```diff
@@ -856,8 +856,7 @@ MFNavierStokesPreconditionGMGBase<dim>::reinit(
             level_constraints[level],
             quadrature_mg,
             forcing_function,
-            this->simulation_parameters.physical_properties_manager
-              .get_kinematic_viscosity_scale(),
+            this->simulation_parameters.physical_properties_manager,
```

Comment:
Since you are passing the pointer to it, which is just fine really, every level has the same physical property manager, however, in the code that follows you don't seem to be aware of that.